pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -343,12 +351,8 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication {
         // solhint-disable-next-line not-rely-on-time
         require(block.timestamp >= scheduledExecution.executableAt, "ACTION_NOT_EXECUTABLE");
         if (scheduledExecution.protected) {
-            bool isAllowed = hasPermissionOrWhatever(
-                EXECUTE_ACTION_ID,
-                msg.sender,
-                address(this),
-                bytes32(scheduledExecutionId)
-            );
+            bytes32 executeScheduledActionId = getActionId(EXECUTE_ACTION_ID, bytes32(scheduledExecutionId));
+            bool isAllowed = hasPermission(executeScheduledActionId, msg.sender, address(this));
```

Comment:
This is not the same as the old version, is it? With this change it is not possible to have global execution rights

Reply:
(I don't think we needed those, just pointing that out)